Example/DKHelperTests/NSStringTests.swift

```diff
@@ -176,11 +211,41 @@ extension NSStringTests {
 		XCTAssert(email.isNumeric == false)
 	}
 
+	func test_ShouldVerifyValidNegativeIntegerString() {
+		let email = "-4356789"
+		XCTAssert(email.isNumeric == true)
+	}
+
 	func test_ShouldVerifyValidIntegerString() {
 		let email = "4356789"
 		XCTAssert(email.isNumeric == true)
 	}
 
+	func test_ShouldVerifyValidNegativeDecimalString() {
+		let email = "-34.2476"
+		XCTAssert(email.isNumeric == true)
+	}
+
+	func test_ShouldNotVerifyValidNegativeDecimalStringWithNoAfterZeroValue() {
+		let email = "-34."
+		XCTAssert(email.isNumeric == false)
```

Comment:
How about ```XCTAssertFalse(email.isNumeric == true)```
I'm not sure if it makes sense in this case since we're using that stupid ```== bool``` but usually you should use
XCTAssertFalse([expression]) when you expect a test to fail